docker/lib/dependabot/docker/update_checker.rb

```diff
@@ -43,9 +43,9 @@ module Dependabot
   module Docker
     class UpdateChecker < Dependabot::UpdateCheckers::Base
       VERSION_REGEX =
-        /v?(?<version>[0-9]+(?:(?:\.[_a-z0-9]+)|(?:-(?:kb)?[0-9]+))*)/i
-      VERSION_WITH_SFX = /^#{VERSION_REGEX}(?<suffix>-[a-z0-9.\-]+)?$/i
-      VERSION_WITH_PFX = /^(?<prefix>[a-z0-9.\-]+-)?#{VERSION_REGEX}$/i
+        /v?(?<version>[0-9]+(?:\.[_a-z0-9]+|-(?:kb)?[0-9]+)*)/i
```

Comment:
```
      VERSION_REGEX = /v?(?<version>[0-9]+(?:\.[_a-z0-9]+|-(?:kb)?[0-9]+)*)/i
```

No need for the newline now that it's shorter, and the newline is making it more painful to review than it needs to be... 

Unfortunately I can't do this as a `suggestion` since it thinks I'm trying to replace other lines too...

Reply:
Happy to remove the newline but I don't understand how introducing less changes makes the PR harder to review? Is it because the new line distracts you? To me limiting the diff to what I want to fix is helpful, GitHub also displays in darker green or red the specific changes:

<img width="859" alt="Captura de pantalla 2022-11-08 a las 21 21 30" src="https://user-images.githubusercontent.com/2887858/200667736-099a57ba-a034-4ad7-b6b2-e84b501eba23.png">

Anyways, let me remove the new line now. This kind of artificial line breaks is why I never set any hard limits on line length.

Reply:
To complete my point, this is the new diff, in my eyes it's harder to review

<img width="847" alt="Captura de pantalla 2022-11-08 a las 21 26 38" src="https://user-images.githubusercontent.com/2887858/200668379-90ba77ac-e29f-4754-993f-d7c49f1e4774.png">

Reply:
* I was using `diff view: split` setting
* I wasn't so much reviewing the delta between old/new, as reviewing the flow of how the regexes chain their inheritance to be sure the new code's logic made sense... having them stacked vertically such that all three lines begin at nearly the same column makes it easier.

Reply:
Yeah, I forget the "unified" setting even exists because I use it so rarely... the split diff allows me to trace new code flow w/o getting interuptted by the dead code... here's what I see now:

![image](https://user-images.githubusercontent.com/483314/200669199-4ae330f6-4512-4f82-90fc-95f92c462770.png)